.github/workflows/generate-documentation.yml

```diff
@@ -0,0 +1,28 @@
+name: Generate documentation
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: 
+      - master
+  push:
+    branches: 
+      - master
+jobs:
+  generate_doc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Install dependencies
+        run: npm install
+      - name: Build docs
+        run: npm run build
```

Comment:
in #5, I suggested to run the script that generate the site for local evaluation without server. This one requires an http server to be fully functionnal (links generated by Antora won't contain html extension for instance)
Notice that this will require to build in a different way when we target the gh-pages environment.

Reply:
fixed with 46a28fe